tests/unit/utils/test_models.py

```
@@ -0,0 +1,21 @@
+
+def test_shorten_1_part():
+    shortened = get_shortened_model_name("foo")
+    assert shortened == "foo"
+
+
+def test_shorten_3_parts():
+    shortened = get_shortened_model_name("models.foo.bar")
+    assert shortened == "bar"
+
+
+def test_shorten_1_part():
```

Flake8 (7.3.0)

[error] redefinition of unused 'test_shorten_1_part' from line 9

(F811)